src/codecs/mvt/Category.js

```diff
@@ -2,15 +2,15 @@ import CategoryCodec from '../Category';
 import CartoRuntimeError, { CartoRuntimeTypes as crt } from '../../errors/carto-runtime-error';
```

Comment:
You can change the alias of the import to 'errors' for consistency with your changes in other files. 
I don't care much about the alias itself, but it's nicer if we are consistent with them all across the library.

Reply:
You're right, I didn't realize 👍